packages/core/plugin/src/withSentryAndroidGradlePlugin.ts

```diff
@@ -0,0 +1,96 @@
+import { withAppBuildGradle, withProjectBuildGradle } from '@expo/config-plugins';
+
+export interface SentryAndroidGradlePluginOptions {
+  sentryAndroidGradlePluginVersion?: string;
```

Comment:
Let's bake in a version that's compatible with current RN SDK and keep it our responsibility instead of users having to pick SAGP version.

Reply:
Updated with 2539a24acbf60729e8b56f18ab2ab8c372a76945 to hardcode the SAGP version and use `enableAndroidGradlePlugin` as a flag to enable.